widgets/common/src/DeploymentActions.js

```diff
@@ -82,14 +82,18 @@ export default class DeploymentActions {
     }
 
     doSetSite(deploymentId, siteName, detachSite) {
-        const data = { detach_site: detachSite };
-        if (siteName) {
-            data.site_name = siteName;
-        }
+        const data = detachSite ? { detach_site: detachSite } : { site_name: siteName };
 
         return this.toolbox.getManager().doPost(`/deployments/${deploymentId}/set-site`, null, data);
     }
 
+    doGetSite(deploymentId) {
```

Comment:
A more suitable name IMO would be

```suggestion
    doGetSiteName(deploymentId) {
```

Reply:
Changed.